src/main.ts

```diff
@@ -4,7 +4,7 @@ import { registerSwagger } from "@config/swagger.ts";
 import cors from "@fastify/cors";
 import helmet from "@fastify/helmet";
 import jwt from "@fastify/jwt";
-import routes from "@routes/index.ts";
+import routes from "@src/modules/index.ts";
```

Comment:
rename @routes -> @modules on `tsconfig.json`.
```suggestion
import routes from "@modules/index.ts";
```